app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>SUPER ROBOT</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+  <link href='https://fonts.googleapis.com/css?family=Anton|Open+Sans:400,300,700,300italic,400italic,700italic' rel='stylesheet' type='text/css'>
```

Comment:
Rails might take care of this automatically, but in general I would place this link to a Google font before the stylesheet that uses it.